sdk/servicebus/Azure.Messaging.ServiceBus/tests/Core/ServiceBusRetryPolicyTests.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="ServiceBusRetryPolicy" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class ServiceBusRetryPolicyTests
+    {
+        [Test]
+        public void HasDefaultValues()
+        {
+            var policy = new MockServiceBusRetryPolicy();
+
+            Assert.That(policy.IsServerBusy, Is.False);
+            Assert.That(policy.ServerBusyBaseSleepTime, Is.EqualTo(TimeSpan.FromSeconds(10)));
```

Comment:
I wonder if this may be too specific.   Should we consider just a check for > 0?

Reply:
Will change